soc/nordic/nrf54h/power.c

```diff
@@ -33,7 +33,6 @@ static void suspend_common(void)
 
 	/* Disable retention */
 	nrf_lrcconf_retain_set(NRF_LRCCONF010, NRF_LRCCONF_POWER_DOMAIN_0, false);
-	nrf_lrcconf_poweron_force_set(NRF_LRCCONF010, NRF_LRCCONF_POWER_DOMAIN_0, false);
```

Comment:
This change looks unrelated, and neither change has a comment that marks it as workaround or temporary when a better solution is in place